spec_creation/autofill_eval_spec.py

```diff
@@ -285,6 +285,11 @@ def validate_and_fill_eval_trips(curr_spec):
     modified_spec = copy.copy(curr_spec)
     eval_trips = modified_spec["evaluation_trips"]
     for t in eval_trips:
+
+        # key that holds information regarding timespan of ground truth data
+        # TODO: determine where to find timespan info for trip
+        t["timespan"] = ""
```

Comment:
This is roughly what I had in mind. But I think that the timespan needs to be at the section/leg level, and not the trip level. The commit I made yesterday has an example of how a user might specify the valid timespan.

https://github.com/MobilityNet/mobilitynet-analysis-scripts/commit/d647389ab3a5786fea61a60dd5b7902c70d170e9#diff-babc6ee3762947af73e5aaae138f9c22badd4b75cd4c84b06d7cd0017ac36f62R397

So there are two "Wurster bikeshare station" entries and each of them is valid for some time range.
Note that the "Wurster bikeshare station" entries are in the *leg* and not the *trip* because the rest of the trip is not necessarily modified.

Reply:
But the copy code seems like it would be fairly similar

Reply:
Added a new commit, let me know if my updates reflect what you have in mind!